css/fonts.css

```diff
@@ -0,0 +1,40 @@
+
+@font-face {
+ font-family: "Montserrat";
+ src: url("../fonts/MontserratAlternates-Bold.eot"), /* IE9 Compat Modes */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5516260)